### PR TITLE
fix: restore backward compatibility with Clappr 0.2.X

### PR DESCRIPTION
**Pull request by kslimani**
Fix backward compatibility with Clappr 0.2.X.

Fix a minor issue with some events trigger twice while reloading ui.

Added Dash shaka playback plugin in dev. test page to ease future debug.

PS: some "chore" improvement could be also to remove Clappr from dev dependencies. _(now loaded from jsDelivr on test page)_

### CODE REVIEW

**Comment by leandromoreira** [comment]
Thank you so much @kslimani your care and efforts are what keeps open source community alive and kicking

**Comment by kslimani** [comment]
Added a minor fix to make the selector scrollable when too many levels to display.

File: src/main.js [hunk 1]
```diff
@@ -25,49 +25,55 @@ export default class LevelSelector extends UICorePlugin {
     }
   }
 
+  get __container() {
```
Comment (joaopaulovieira) [inline src/main.js]: We can just call this getter as `container`, no?

File: src/main.js [hunk 2]
```diff
@@ -25,49 +25,55 @@ export default class LevelSelector extends UICorePlugin {
...
+    return this.core.activeContainer
+      ? this.core.activeContainer
+      : this.core.mediaControl.container
+  }
+
+  get __playback() {
```
Comment (joaopaulovieira) [inline src/main.js]: We can just call this getter as `playback`, no?

File: src/main.js [hunk 3]
```diff
@@ -25,49 +25,55 @@ export default class LevelSelector extends UICorePlugin {
...
+    })
   }
 
   shouldRender() {
-    if (!this.core.activeContainer) return false
+    if (!this.__container) return false
 
-    let currentPlayback = this.core.activePlayback
-    if (!currentPlayback) return false
+    if (!this.__playback) return false
```
Comment (joaopaulovieira) [inline src/main.js]: We can check this in one line, what you think?

```suggestion
    if (!this.__playback || !this.__container) return false
```

File: src/main.js [hunk 4]
```diff
@@ -81,13 +87,23 @@ export default class LevelSelector extends UICorePlugin {
...
+      this.$('.level_selector ul').css('max-height', this.core.el.offsetHeight*0.8)
       this.highlightCurrentLevel()
     }
     return this
   }
 
   fillLevels(levels, initialLevel = AUTO) {
     if (this.selectedLevelId === undefined) this.selectedLevelId = initialLevel
+
+    let onLevelsAvailable = this.core.options.levelSelectorConfig.onLevelsAvailable
```
Comment (joaopaulovieira) [inline src/main.js]: Maybe, we need to add more resilience here? Like:
```suggestion
    let onLevelsAvailable = this.core.options && this.core.options.levelSelectorConfig && this.core.options.levelSelectorConfig.onLevelsAvailable
```

File: src/main.js (hunk 1 above)
Comment (kslimani) [reply src/main.js]: Sure, it was prefixed by default to distinguish it as "special" getter; let's refactor them.

**Comment by kslimani** [comment]
Applied changes.

**Pull request merged**